packages/app/package.json

```diff
@@ -65,7 +65,7 @@
 		"@getalby/lightning-tools": "^5.0.3",
 		"@iconify-json/bitcoin-icons": "^1.2.0",
 		"@internationalized/date": "^3.5.4",
-		"@nostr-dev-kit/ndk": "2.10.7",
+		"@nostr-dev-kit/ndk": "2.10.6",
```

Comment:
why rollback?

Reply:
Problems with the ndk and ndkSvelte packages, it seems that they have not dumped the versions...